src/main/deltachat/controller.ts

```diff
@@ -166,6 +172,29 @@ export default class DeltaChatController extends EventEmitter {
         logCoreEvent.debug(event, data1, data2)
       }
 
+      // Network Status
+      if (event === 'DC_EVENT_ERROR_NETWORK') {
+        this.networkStatus = false
+        this.networkStatusMessage = data1 + data2
```

Comment:
https://github.com/deltachat/deltachat-core-rust/blob/master/src/events.rs#L81-L95

either data 1 or data 2 seems unused, but I didn't look fully into how the core works here.